calfem/mesh.py

```diff
@@ -1010,7 +1010,7 @@ def trimesh2d(vertices, segments=None, holes=None, maxArea=None, quality=True, d
 
     filename = "./trimesh.temp/polyfile.poly"
 
-    if not segments is None:
+    if not (segments is None):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Use `is not` operator rather than `not ... is` because it is more readable and makes your code easier to understand.

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=Use%20is%20not)

Similar issue at line number 1016.